FestivalPlanner/films/views.py

```diff
@@ -323,45 +426,7 @@ def get_submitted_name(request, submit_names):
     return submitted_name
 
 
-def update_rating(session, film, fan, rating_value):
-    old_rating_str = fan.fan_rating_str(film)
-    new_rating, created = FilmFanFilmRating.film_ratings.update_or_create(
-        film=film,
-        film_fan=fan,
-        defaults={'rating': rating_value},
-    )
-    zero_ratings = FilmFanFilmRating.film_ratings.filter(film=film, film_fan=fan, rating=0)
-    if len(zero_ratings) > 0:
-        zero_ratings.delete()
-    init_rating_action(session, old_rating_str, new_rating)
-
-
-def init_rating_action(session, old_rating_str, new_rating):
-    new_rating_name = get_rating_name(new_rating.rating)
-    rating_action = {
-        'fan': str(current_fan(session)),
-        'old_rating': old_rating_str,
-        'new_rating': str(new_rating.rating),
-        'new_rating_name': new_rating_name,
-        'rated_film': str(new_rating.film),
-        'rated_film_id': new_rating.film.id,
-        'action_time': datetime.now().isoformat(),
-    }
-    session[rating_action_key(session)] = rating_action
-
-
-def refresh_rating_action(session, context):
-    key = rating_action_key(session)
-    if key in session:
-        action = session[key]
-        context['action'] = action
-        context['action']['action_time'] = datetime.fromisoformat(action['action_time'])
-
-
-def rating_action_key(session):
-    return f'rating_action_{current_festival(session).id}'
-
-
+@skip("View can only be called from the film ratings view, which initializes the cache data")
```

Comment:
Please remove this test decorator.

When needed, move it to the test code in question.

Reply:
Done.